webapi/src/test/scala/org/knora/webapi/responders/v2/OntologyResponderV2Spec.scala

```diff
@@ -3136,5 +3141,342 @@ class OntologyResponderV2Spec extends CoreSpec() with ImplicitSender {
                     anythingLastModDate = newAnythingLastModDate
             }
         }
+
+        "not load an ontology containing a class that's missing a cardinality for a link value property" in {
+            val invalidOnto = List(RdfDataObject(
+                path = "_test_data/responders.v2.OntologyResponderV2Spec/missing-link-value-cardinality-onto.ttl", name = "http://www.knora.org/ontology/invalid"
+            ))
+
+            loadTestData(invalidOnto)
+
+            expectMsgPF(timeout) {
```

Comment:
As a tip, you could write
```
expectMsgPF(timeout) {
                 case msg: akka.actor.Status.Failure => msg.cause.isInstanceOf[InconsistentTriplestoreDataException] should ===(true)
             }
```

as

```
expectMsgType[akka.actor.Status.Failure](timeout).cause should be (true)
```

`expectMsgPF` is useful when you expect more than one message type.

Reply:
Thanks, I'll do that next time. :)